Xamarin.Essentials/Launcher/Launcher.android.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Android.Content;
+using Android.Content.PM;
+using AndroidUri = Android.Net.Uri;
+using Uri = System.Uri;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Launcher
+    {
+        static Task<bool> PlatformCanOpenAsync(Uri uri)
+        {
+            var intent = CreateIntent(uri.ToString());
```

Comment:
should this be uri.ToString?  Should it be the actual AbsolutePath?

Reply:
You are correct, I read through the microsoft docs and changed it. AbsolutePath should always be used when you want an escaped string, and we do. I also changed it on iOS. UWP needs an uri either way so it will work there.